### ISSUE TITLE
Colleagues don't all display on profile

### ISSUE THREAD

**Issue by klecuyer** [gctools-outilsgc/gccollab#314]
As a user, I want to see how many colleagues I have and be able to interact with them.

For some reason, when I look at my profile, I see only 10 colleagues: https://gccollab.ca/profile/Ashleyevans
![image](https://user-images.githubusercontent.com/29754185/38687980-f5a0108c-3e45-11e8-93cb-4e7ec9989c9e.png)

But when I click to see a full list, I actually have over 100 colleagues: https://gccollab.ca/friends/Ashleyevans
![image](https://user-images.githubusercontent.com/29754185/38688046-20f98bb4-3e46-11e8-9b12-06a9c35b4bb0.png)

**Comment by markwooff** [comment]
By default, the list of "My Colleagues" would only show 10 users. I've increased this to 14 to fill out the column and the # of colleagues should now be correct.

**Comment by klecuyer** [comment]
@markwooff just had a user report this in GCconnex. Can we pull this fix over easily?

If not I'll create a new ticket.
![image](https://user-images.githubusercontent.com/29754185/38745848-c8af1ab8-3f13-11e8-930a-8bd6c026b4cb.png)

**Comment by markwooff** [comment]
The fix should be applied to GCconnex as well whenever the next code push is done 👌

**Comment by klecuyer** [comment]
It looks like the colleagues display properly on a user's profile now. @markwooff can you speak to the way the user's colleagues display on their colleagues page? Specifically if they have over 700 colleagues.
You can see it in action here: https://gcconnex.gc.ca/friends/Ainslie.Nolan

It looks like it displays the first 700 in 10 per page (70 pages) but there's another separate page navigation for the remainder
![image](https://user-images.githubusercontent.com/29754185/38952102-8515ea80-4318-11e8-8704-4842099fbc9d.png)
![image](https://user-images.githubusercontent.com/29754185/38952109-8c3a7696-4318-11e8-911c-694b0978df6c.png)

*edit I know this example is in GCconnex but I understand the fix was pushed in both connex and collab

**Comment by markwooff** [comment]
Yeah this is separate from the previous issue, but I can say that the pagination for that section is a little wonky because the first row of pagination is because the users are displays in "data tables". When you click on those numbers, the list above reloads instantly. Clicking on the second row brings in a whole new batch of users that get loaded into the data table for the same functionality.

I imagine this was done because it would be very slow to load in 700+ users into a data table. I would advise creating a separate issue for this and have someone with more knowledge on the subject take a look 👍
